src/framework/core/js/FluidDocument.js

```diff
@@ -174,4 +174,10 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         };
     });
 
+    // Sets the value to the node and fires the change event
```

Comment:
Better docs would be helpful here - up to the standard of the JSDoc-style comments that are the basis of the newer-style documentation - indicating i) the types and names of the arguments, ii) any return value, iii) what the function actually does - "fires the change event" is too vague to allow the user to interpret the meaning of this function, even though in theory it is pretty obvious